src/storybook.js

```diff
@@ -155,6 +155,46 @@ var staticServer = exports.staticServer = function(config, options, callback) {
   }).catch(callback);
 };
 
+var setStorybookConfig = exports.setStorybookConfig = function(storybookApp, storybookVersion, storybookConfigDir) {
+  var configPath = path.resolve(process.cwd(), storybookConfigDir, 'config.js');
+  var isNewFile = false;
+  if (!fs.existsSync(configPath)) {
+    // handle declaritive configuration and preview.js in Storybook 5.3+
```

Comment:
@loyalchow can we add this link to the comment so people have context on the change? https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#from-version-52x-to-53x